docs/examples/hello-world/src/main.rs

```diff
@@ -1,16 +1,11 @@
 use via::{Next, Request, Result};
 
-async fn hello(mut request: Request, _: Next) -> Result<String> {
-    // Attempt to parse the query parameter `n` to a `usize`. If the query
-    // parameter doesn't exist or can't be converted to a `usize`, default to 1.
-    let n = request.query("n").first().parse().unwrap_or(1);
```

Comment:
This was a fun way to test out the params / query params API. I didn't mean to commit it.